addons/fluentbit/fluentbit.yaml

```
@@ -177,7 +177,7 @@ spec:
               Time_Key @ts
               Logstash_Format On
               Logstash_Prefix kubernetes_audit
-              Retry_Limit False
+              Retry_Limit 10
```

I think we'll need to set this to a low number. The backoff is pretty aggressive, chunks that will never succeed can quickly get into several minutes between retries, and there are enough of them that this can easily cause audit log collection to stop. I found that setting this to 1 on soak worked well. Let's also add a comment explaining why we need to drop audit logs more aggressively.

```suggestion
              # A small fraction of audit logs will be rejected by Elasticsearch because values may have an inconsistent type.
              # We set a low retry limit here in order to prevent fluent-bit from backing up while it repeatedly retries flushing these logs.
              # See https://github.com/uken/fluent-plugin-elasticsearch#random-400---rejected-by-elasticsearch-is-occured-why.
              Retry_Limit 1
```